lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/nested-journey-definitions.yaml

```diff
@@ -195,7 +195,7 @@ WEB_DL_OR_PASSPORT:
           targetState: PROVE_ANOTHER_WAY_AFTER_PASSPORT
           checkIfDisabled:
             f2f:
-              targetState: MULTIPLE_DOC_CHECK_PAGE
+              exitEventToEmit: access-denied-passport-or-dl
```

Comment:
There's an interesting inconsistency here, depending on when exactly F2F is disabled 🤔 

Most of the time it'll be caught here and you'll go -> MULTIPLE_DOC_CHECK_PAGE
But if it's slightly later you'll go -> PROVE_ANOTHER_WAY_AFTER_PASSPORT -> PYI_ESCAPE

I wonder if we can unify them (and potentially just drop one of them), i.e.:
- Make this also emit `end`
- Make the `end` event go to `MULTIPLE_DOC_CHECK_PAGE` if F2F is disabled

What do you think? We could potentially check with Joe U when he's back, but it seems pretty low-stakes!

Reply:
Yeah that's true. I think the best thing for the user, without changing too much stuff. is to drop the f2f disabled check on the access-denied event and allow the user to progress to `PROVE_ANOTHER_WAY_AFTER_PASSPORT`. The working might be a little confusing but it will give them two options -> DL CRI or "try using another type of photo ID". If they choose the other way (f2f) then catch that and send them to the escape page. 
Sending them back to the multidoc page will also have slightly confusing wording, but gives them two options that won't work instead of one...

Reply:
Actually, I just remembered how the user gets `access-denied` from passport, so it won't be confusing at all if they get the `PROVE_ANOTHER_WAY_AFTER_PASSPOR` page.